deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -1,7 +1,23 @@
 const sanitizeFilename = require("sanitize-filename");
+const { resolveFundamental } = require("@mdn/fundamental-redirects");
+const { DEFAULT_LOCALE, VALID_LOCALES } = require("@mdn/global-constants");
+const acceptLanguageParser = require("accept-language-parser");
 
 const CONTENT_DEVELOPMENT_DOMAIN = ".content.dev.mdn.mozit.cloud";
 
+const VALID_LOCALES_LIST = [...VALID_LOCALES.values()];
+
+function guessLanguage(request) {
+  // Do we want to support a language cookie? Add it here!
+  // Each header in request.headers is always a list of objects.
+  const acceptLangHeaders = request.headers["accept-language"];
+  const { value = null } = (acceptLangHeaders && acceptLangHeaders[0]) || {};
```

Comment:
@peterbe That won't work as you've suggested, because the result of `acceptLangHeaders[0]` is not the value of the header, but an object `{key: ..., value: ...}`, so it's really a matter of preference between:
```js
  const { value = null } = (acceptLangHeaders && acceptLangHeaders[0]) || {};
```
or 
```js
  const value = (acceptLangHeaders && acceptLangHeaders[0].value) || null;
```

Reply:
that would even have to be:
```js
  const value = (acceptLangHeaders && acceptLangHeaders[0] && acceptLangHeaders[0].value) || null;
```